### PR TITLE
B2b mp vouchers endpoints

### PR DESCRIPTION
**Pull request by lera-coder**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    ...    AND    I set Headers:    Authorization=${token}
+    ...    AND    I send a POST request:    /carts    {"data": {"type": "carts","attributes": {"priceMode": "${GROSS_MODE}","currency": "${currency_code_eur}","store": "${store_de}","name": "${test_cart_name}-${random}"}}}
+    ...    AND    Save value to a variable:    [data][id]    cart_id
+    ...    AND    Response status code should be:    201
+    ...    AND    I send a POST request:    /carts/${cart_id}/items    {"data": {"type": "items","attributes": {"sku": "419901","quantity": 1}}}
+    ...    AND    Response status code should be:    201
+    When I send a POST request:    /carts/${cart_id}/vouchers    {"data": {"type": "vouchers","attributes": {"code": "1111111"}}}
+    Then Response status code should be:    422
+    And Response reason should be:    Unprocessable Content
+    And Response body parameter should not be EMPTY:    [errors]
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot:25]: please check the error text, not just that there are errors

File: resources/environments/environments_api_mp_b2c.json [hunk 2]
```diff
@@ -197,6 +197,12 @@
         "discount_concrete_product_2_total_sum_of_discounts": "8096",
         "discount_concrete_product_3_sku": "401768",
         "discount_2_name": "20% off cameras",
+        "discount_id_3_name": "5% off white",
+	    "discount_id_3_description": "Get a 5% discount on all white products with voucher code",
+        "discount_id_3_voucher_code": "sprykercu7r",
+        "discount_id_4_name": "10% off Safescan",
+	    "discount_id_4_description": "Get a 10% discount on all Safescan brand products with voucher code",
+        "discount_id_4_voucher_code": "safescanda2wa4ju8f",
```
Comment (OKrivtsova) [inline resources/environments/environments_api_mp_b2c.json:205]: why do you change this environment file if you add tests for b2b mp

Comment (lera-coder) [reply resources/environments/environments_api_mp_b2c.json:205]: @OKrivtsova changed.

File: tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot (hunk 1 above)
Comment (lera-coder) [reply tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot:25]: @OKrivtsova done.

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**